sneck/Deck.py

```diff
@@ -809,6 +831,16 @@ def get_id(self) -> int:
     def get_tag(self) -> str:
         return self.__tag
 
+    def set_title(self, title: str) -> None:
+        self.__title = title
+
+    def set_order(self, order: int) -> None:
+        self.__order = order
+
+    def set_deletion_time(self, deletion_time: Union[dt, int]) -> None:
```

Comment:
Deletion time is not a property that is meant to be user settable, but rather a read-only information that the Deck provides us.